### ISSUE TITLE
Calculation for estimated storage is incorrect

### ISSUE THREAD

**Issue by mtlynch** [NebulousLabs/Sia-UI#775]
Sia-UI estimates the storage capacity based on the allowance:

![image](https://user-images.githubusercontent.com/7783288/37778686-ecd6c5d0-2dc0-11e8-8377-d40e7e952fa0.png)

The calculation is here:

https://github.com/NebulousLabs/Sia-UI/blob/e6be3f69901a8e548be42ab83642e440c2277d33/plugins/Files/js/sagas/files.js#L50-L52

This calculation is essentially:

```
[allowance] / [storage_per_tb_per_month]
```

This is incorrect as it neglects upload cost and contract fees. It also misuses the storage price as the price per TB per **contract period** when it is actually the price per TB per **month**. The actual calculation requires Sia-UI to solve for `tb_count` in this equation:

```
allowance = (storage_per_tb_per_month * tb_count * months_per_contract) +
  (upload_per_tb * tb_count) +
  (contract_fees)
```

**Comment by DavidVorick** [comment]
don't know how this slipped through for so many months, this is an easy fix. @eddiewang

**Comment by eddiewang** [comment]
Thanks for the equation @mtlynch - i went ahead and implemented that into PR #800. Feel free to give it a glance to see if it makes sense. From my testing, the numbers look right.
